Sources/StreamChat/APIClient/Endpoints/Payloads/MessagePayloads.swift

```diff
@@ -94,6 +96,8 @@ class MessagePayload: Decodable {
     var pinnedBy: UserPayload?
     var pinnedAt: Date?
     var pinExpires: Date?
+    
+    var poll: PollDTO?
```

Comment:
DTOs should not be in Payloads. Why is this here?

Reply:
yeah, I was mostly following the REST API naming - let me fix those.